experimental/uefi/app/src/main.rs

```diff
@@ -71,6 +73,8 @@ fn main(handle: Handle, system_table: &mut SystemTable<Boot>) -> Status {
     serial_echo(handle, system_table.boot_services(), ECHO_SERIAL_PORT_INDEX).unwrap();
 }
 
+const MOCK_SESSION_ID: [u8; 8] = [0; 8];
+
```

Comment:
Mock for now as we only have a single proof of concept client. Will swap this out once I amend the client code to support attestation